### PR TITLE
example of saving and loading trials

### PR DESCRIPTION
**Pull request by bjkomer**
Adding an example to the docs of how to save the trials object and continue the search from where it was left off. Addresses #267

### CODE REVIEW

File: docs/templates/getting-started/minimizing_functions.md [hunk 1]
```diff
@@ -181,4 +214,4 @@ It's normal if this doesn't make a lot of sense to you after this short tutorial
 but I wanted to give some mention of what's possible with the current code base,
 and provide some terms to grep for in the hyperopt source, the unit test,
 and example projects, such as [hyperopt-convnet](https://github.com/jaberg/hyperopt-convnet).
```
Comment (maxpumperla) [inline docs/templates/getting-started/minimizing_functions.md]: I know this is not from your PR, but would you please replace all `jaberg` references by `hyperopt`? thanks!

**Comment by maxpumperla** [comment]
@bjkomer thank you! this is much needed. just the little, unrelated nit and we're good to go here

File: docs/templates/getting-started/minimizing_functions.md (hunk 1 above)
Comment (bjkomer) [reply docs/templates/getting-started/minimizing_functions.md]: Good idea, they've been replaced now

**Comment by maxpumperla** [comment]
@bjkomer awesome, thanks!

**Comment by wxie2013** [comment]
where is the conclusion of all these discussion documented? The website: https://hyperopt.github.io/hyperopt/,  seems outdated.

**Comment by wxie2013** [comment]
> where is the conclusion of all these discussion documented? The website: https://hyperopt.github.io/hyperopt/, seems outdated. By combing though gitbub, the latest doc seems to be the following which is dated 12/18/2019
> 
> https://github.com/bjkomer/hyperopt/blob/e017ddc60331631c7fc4a9208f1b3a2794849e88/docs/templates/getting-started/minimizing_functions.md
> 
> while in the official doc, it's dated Feb 2018.

also for the case the continuing the training from previous run, above threshold mention that one need to load the same rstate object while in the 12/18/2019 doc, it doesn't say so.  What should one do in this case? Maybe it doesn't matter to use either case because the starting point of the tries is the same?

**Pull request merged**